src/clusterfuzz/_internal/tests/appengine/handlers/cron/triage_test.py

```diff
@@ -388,3 +388,12 @@ def test_unrecovered_exception(self):
     testcase = data_handler.get_testcase_by_id(self.testcase.key.id())
     self.assertEqual('Failed to file issue due to exception: unrecovered',
                      testcase.get_metadata(triage.TRIAGE_MESSAGE_KEY))
+
+  def test_no_experimental_bugs(self):
+    """Test no exception."""
```

Comment:
nit: "Tests"
And I'm not sure this description is correct.

Reply:
fixed!